src/script/campaign/mod/vco-listeners.lua

```diff
@@ -2,8 +2,27 @@ local vco = core:get_static_object("vco");
 
 -- COMMON --
 
-local function earned_x_pooled_resource(resource, amount)
-  return true
+local function faction_earned_x_pooled_resource(resource, amount)
```

Comment:
This more a professional note than a real requirement, but I prefer something that describes an action for the name of a function. I think `faction_earned_x_pooled_resource` sould like the name of an event. I think something like `update_faction_pooled_resource_gains` or similar.